src/main/java/com/birdie/kafka/connect/smt/JsonSchema.java

```diff
@@ -0,0 +1,92 @@
+package com.birdie.kafka.connect.smt;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.data.*;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.apache.kafka.connect.transforms.util.SimpleConfig;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URISyntaxException;
+import java.util.Map;
+
+public class JsonSchema implements Transformation<SourceRecord> {
```

Comment:
Let's be explicit in the naming.

```suggestion
public class DebeziumJsonDeserializer implements Transformation<SourceRecord> {
```